transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java

```diff
@@ -111,13 +111,26 @@ public EmbeddedChannel(boolean hasDisconnect, ChannelHandler... handlers) {
     }
 
     /**
+     * Create a new instance with the pipeline initialized with the specified handlers.
+     *
+     * @param register {@code true} if this {@link Channel} is registered to the {@link EventLoop} in the
+     *                 constructor. If {@code false} the user will need to call {@link #register()}.
+     * @param hasDisconnect {@code false} if this {@link Channel} will delegate {@link #disconnect()}
+     *                      to {@link #close()}, {@link false} otherwise.
+     * @param handlers the {@link ChannelHandler}s which will be add in the {@link ChannelPipeline}
+     */
+    public EmbeddedChannel(boolean register, boolean hasDisconnect, ChannelHandler... handlers) {
```

Comment:
we cant as this clash with the existing constructor:

https://github.com/netty/netty/blob/66d82a22ea4dac414a101993342d93878a7e5ea2/transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java#L109